pkg/testing/define/define_define.go

```diff
@@ -0,0 +1,28 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+//go:build define
+
+package define
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+)
+
+func defineAction(t *testing.T, req Requirements) *Info {
+	// always validate requirement is valid
+	if err := req.Validate(); err != nil {
+		panic(fmt.Sprintf("test %s has invalid requirements: %s", t.Name(), err))
+	}
+	// skip recording the requirements for the test
```

Comment:
Can you reword this please? For a few minutes I was reading it as "we're going to skip recording the requirements for the test" when I think what it actually means is something more like "we're going to be using `t.Skip` to record the requirements for the test".

Reply:
Fixed.